.pre-commit-config.yaml

```diff
@@ -0,0 +1,5 @@
+repos:
+-   repo: https://github.com/awslabs/git-secrets
+    rev: 80230afa8c8bdeac766a0fece36f95ffaa0be778
+    hooks:
+        - id: git-secrets
```

Comment:
Have you considered adding any other pre-commit hooks (i.e. linters, etc.)? I'm betting that a lot of committers will forget to run/install pre-commit locally (especially if developers work across multiple environments) even if you add it to your documentation. Adding more other hooks might make it more likely for committers to accidentally trip up the CI-side pre-commit check (e.g. with forgetting a newline at the end of a Python file) and force them to install pre-commit locally. Make sense?

Reply:
Thank you, yes, I will add more checks for it.